packages/terra-collapsible-menu-view/src/terra-dev-site/test/collapsible-menu-view/DefaultCollapsibleMenuView.test.jsx

```diff
@@ -71,6 +71,7 @@ class CollapsibleMenuViewDemo extends React.Component {
           text="Menu Button 1"
           key="MenuButton1"
           className="MenuButton1"
+          onClick={this.handleToggleOneOnChange}
```

Comment:
can we have different event created for this example ? like handleMenuButtonOnClick ? to be more clear on what we are trying to do.

Reply:
Done b04c426